shared/js/background/chrome-events.es6.js

```diff
@@ -271,6 +274,55 @@ chrome.webRequest.onBeforeSendHeaders.addListener(
     ['blocking', 'requestHeaders']
 )
 
+/*
+ * Truncate the referrer header according to the following rules:
+ *   Don't modify the header when:
+ *   - If the header is blank, it will not be modified.
+ *   - If the referrer domain OR request domain are safe listed, the header will not be modified
+ *   - If the referrer domain and request domain are part of the same entity (as defined in our
+ *     entities file for first party sets), the header will not be modified.
+ *
+ *   Modify the header when:
+ *   - If the destination is in our tracker list, we will trim it to eTLD+1 (remove path and subdomain information)
+ *   - In all other cases (the general case), the header will be modified to only the referrer origin (includes subdomain).
+ */
+let ReferrerListenerOptions = ['blocking', 'requestHeaders']
```

Comment:
nit - this should be in the linter, but I'd avoid capitalizing variables that are not classes